### PR TITLE
update contrib guide

### PR DESCRIPTION
**Pull request by navalev**

### CODE REVIEW

**Review by shiranr — APPROVED** [review]

**Comment by omri374** [comment]
Looks great. One minor comment. The intro says "It's also a system you could take as a whole or in parts and deploy locally, on-prem on in the cloud, especially when leveraging Kubernetes.". Maybe we should remove the Kubernetes mention?

**Review by omri374 — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,41 +23,27 @@ To get started, refer to the documentation for [setting up a development environ
...
 ### How can I contribute?
 
-- [Testing](#how-to-test)
-- [Adding new recognizers for new PII types](#adding-new-recognizers-for-new-pii-types)
-- [Adding new connectors](#adding-new-connectors)
-- [Bug fixing and general improvement](#bug-fixing-and-general-improvement)
-
+-   [Testing](#how-to-test)
+-   [Adding new recognizers for new PII types](#adding-new-recognizers-for-new-pii-types)
+-   [Bug fixing and general improvement](#bug-fixing-and-general-improvement)
```
Comment (TessFerrandez) [inline CONTRIBUTING.md:36]:
```suggestion
-   [Fixing Bugs and improving the code](#fixing-bugs-and-improving-the-code)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,41 +23,27 @@ To get started, refer to the documentation for [setting up a development environ
...
 
-Best practices for recognizers development [are described here](docs/developing_recognizers.md). Please follow these guidelines when proposing new recognizers.
-
-### Adding new connectors
-
-Presidio's strength lies in its ability to be completely pluggable and customizable. Adding new types of databases, stream-analytics engines and data stores is extremely useful for many users.
-Use the following [developer guide](docs/tutorial_connector.md) for adding new connectors.
+Best practices for recognizers development [are described here](docs/analyzer/developing_recognizers.md). Please follow these guidelines when proposing new recognizers.
 
 ### Bug fixing and general improvement
```
Comment (TessFerrandez) [inline CONTRIBUTING.md:48]:
```suggestion
### Fixing Bugs and improving the code
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (TessFerrandez) [reply CONTRIBUTING.md:36]: Just a nit to make the list language consistent

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -23,41 +23,27 @@ To get started, refer to the documentation for [setting up a development environ
...
 
 Adding a new recognizer is a great way to improve Presidio. A new capability to detect a new type of PII entity improves Presidio's coverage and makes private data less accessible.
 
-Best practices for recognizers development [are described here](docs/developing_recognizers.md). Please follow these guidelines when proposing new recognizers.
-
-### Adding new connectors
-
-Presidio's strength lies in its ability to be completely pluggable and customizable. Adding new types of databases, stream-analytics engines and data stores is extremely useful for many users.
-Use the following [developer guide](docs/tutorial_connector.md) for adding new connectors.
+Best practices for recognizers development [are described here](docs/analyzer/developing_recognizers.md). Please follow these guidelines when proposing new recognizers.
```
Comment (TessFerrandez) [inline CONTRIBUTING.md:46]:
```suggestion
Best practices for developing recognizers [are described here](docs/analyzer/developing_recognizers.md). Please follow these guidelines when proposing new recognizers.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -23,41 +23,27 @@ To get started, refer to the documentation for [setting up a development environ
...
-- In order for a pull request to be accepted, the CI (containing unit tests, e2e tests and linting) needs to succeed, in addition to approvals from two maintainers.
-- PRs should be small and solve/improve one issue at a time. If you have multiple suggestions for improvement, please open multiple PRs.
-- Every PR must bump the presidio version by changing the VERSION file to the appropriate [semver](https://semver.org/).
-
-#### Versioning guidelines
-
-- A typical pull request with a new feature bumps the minor version.
-- Bug fix, documentation change and operational code (i.e. devops) change bump the patch version.
-- A breaking change bumps the major version. Please consult with the Presidio team prior to opening such PRs.
+-   A Github issue suggesting the change should be opened prior to a PR.
```
Comment (TessFerrandez) [inline CONTRIBUTING.md:26]: just curious - why was the extra whitespace after the dash added, in this list and next. It doesn't make a difference visually in github, but it deviates from standard markdown syntax.

**Review by TessFerrandez — APPROVED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (navalev) [reply CONTRIBUTING.md:26]: auto indent with vscode

**Pull request merged**